app/views/posts/show.html.erb

```diff
@@ -1,8 +1,14 @@
 <section class="full-post-section">
   <div class="full-post border-black">
-    <h2>Post</h2>
+    <h2>Post # <%= @post.id %> by <%= @post.author.name %></h2>
     <p class="post-likes-comments">Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
     <p><%= @post.text %></p>
+    <div class="like-comment-btn">
+      <%= form_with model: @like, url: user_post_likes_path(user_id: @post.author.id, post_id: @post.id), remote: true do |f| %>
+        <%= f.submit "Add a like", class: "btn" %>
```

Comment:
- [ ] When I add a like, it is not added to the database and 0 likes are returned. This is because, in your Likes model, your foreign key for the User table is `author_id` but your schema says it is `user_id`. After changing it to `user_id`, the likes should be added to the database.

```diff
class Like < ApplicationRecord
- belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+ belongs_to :author, class_name: 'User', foreign_key: 'user_id'
  belongs_to :post
  belongs_to :user

  after_save :update_likes_counter

  private

  def update_likes_counter
    post.increment!(:likes_counter)
  end
end
```